PoseNetYoga/poseyogasite/public/stylesheets/style.css

```diff
@@ -1,21 +1,51 @@
-body {
```

Comment:
Might be better to sort the different css rules. Classes with classes, tags with tags etc.